input/pagecontent/changes.md

```diff
@@ -1,3 +1,14 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01
+- Expected Publication status: TBC
+- Based on FHIR version: 4.0.1
+
+This change log documents the significant updates and resolutions implemented from version 1.0.0-ballot to 1.0.0.
+
+#### Changes in this version
+- Clarify Read/Search Syntax guidance [FHIR-46470](https://jira.hl7.org/browse/FHIR-46740), [FHIR-47194](https://jira.hl7.org/browse/FHIR-47194)
```

Comment:
Please remove, these changes are not substantive.

Reply:
Removed, thanks.